data/json/itemgroups/SUS/domestic.json

```diff
@@ -646,7 +663,11 @@
     "entries": [
       { "distribution": [ { "item": "plunger_toilet", "prob": 90 }, { "item": "plunger_futuristic", "prob": 10 } ] },
       { "item": "brush_toilet", "prob": 75 },
-      { "item": "toilet_paper", "prob": 80 }
+      { "item": "toilet_paper", "prob": 95, "container-item": "wrapper", "charges-min": 1, "charges-max": 1 },
+      { "item": "bathroom_scale", "prob": 50 },
+      { "item": "weeks_old_newspaper", "prob": 5 },
+      { "item": "newest_newspaper", "prob": 2 },
+      { "item": "survnote", "prob": 1 }
```

Comment:
`toilet_paper` doesn't have charges. The others shouldn't be spawning in this SUS group.